src/ykfde-enroll

```diff
@@ -21,7 +24,7 @@ else
   echo "WARNING: Can't access /etc/ykfde.conf. Falling back to defaults."
 fi
 
-while getopts ":d:s:cvh" opt; do
+while getopts ":d:s:kocvh" opt; do
```

Comment:
Those should have the same order as presented below. In that case `":d:s:ckovh"`